jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/java/corrections/proposal/ImplementInterfaceProposal.java

```diff
@@ -34,29 +36,42 @@
 import org.eclipse.jdt.internal.corext.dom.Bindings;
 import org.eclipse.lsp4j.CodeActionKind;
 
-/**
- * QuickFix for implementing interface
- *
- * copied from
- * /org.eclipse.jdt.ui/src/org/eclipse/jdt/internal/ui/text/correction/proposals/ImplementInterfaceProposal.java
- */
-public class ImplementInterfaceProposal extends ChangeCorrectionProposal {
+public class ImplementInterfaceProposal extends ASTRewriteCorrectionProposal {
+
+    private static final String TITLE_MESSAGE = "Let ''{0}'' implement ''{1}''";
```

Comment:
In this case I see the class is used but the change to FilterImplementationQuickFix was to use a message from Messages.java: `Messages.getMessage("LetClassImplement", ...`